clients/include/blas2/testing_tpsv.hpp

```diff
@@ -40,8 +40,9 @@ inline void testname_tpsv(const Arguments& arg, std::string& name)
 template <typename T>
 void testing_tpsv_bad_arg(const Arguments& arg)
 {
-    bool FORTRAN       = arg.api == hipblas_client_api::FORTRAN;
-    auto hipblasTpsvFn = FORTRAN ? hipblasTpsv<T, true> : hipblasTpsv<T, false>;
+    bool FORTRAN          = arg.api == hipblas_client_api::FORTRAN;
+    auto hipblasTpsvFn    = FORTRAN ? hipblasTpsv<T, true> : hipblasTpsv<T, false>;
+    auto hipblasTpsvFn_64 = FORTRAN_64 ? hipblasTpsv_64<T, true> : hipblasTpsv_64<T, false>;
```

Comment:
Well conditional with constant should give warning as I don't see bool FORTRAN_64.   Should check the enum if we did use FORTRAN AND FORTRAN_64, probably best to replace the bool FORTRAN as well.  Both could use the fortran bit mask instead so they are both the same arg.api & c_FORTRAN ?   if same constants used as rocblas.

Reply:
Thanks for pointing it out, I will do the fix now. Sure, @daineAMD could advise about this change.

Reply:
Enum is:
```
typedef enum hipblas_client_api_
{
    C          = 0,
    C_64       = 1,
    FORTRAN    = 2,
    FORTRAN_64 = 3
} hipblas_client_api;
```
I'm fine with `arg.api & c_API_FORTRAN` for both, can change them all afterward.